d2l-user-tile-auto.html

```
@@ -71,7 +67,9 @@
 			if (userUrl && typeof getToken === 'function') {
 				getToken(userUrl)
 					.then(function(token) {
-						self.generateUserRequest(userUrl, token, { background: true });
+						if (typeof token === 'string') {
```

I'm wondering if this check is maybe a bad thing... if the provided function doesn't return a string there will be no feedback to the consumer as to what is going on. It might be better to just pass the value down and if it throws it throws.

That also might apply to the check up on 67 as well. If `getToken` isn't a function maybe just let the javascript explode rather than essentially swallowing up the error condition.